src/server/auth/server/auth_test.go

```diff
@@ -36,16 +37,11 @@ var (
 )
 
 func isAuthActive(tb testing.TB) bool {
-	_, err := seedClient.GetAdmins(context.Background(),
-		&auth.GetAdminsRequest{})
-	switch {
-	case auth.IsErrNotSignedIn(err):
-		return true
-	case auth.IsErrNotActivated(err):
-		return false
-	default:
+	active, err := seedClient.IsAuthActive()
+	if err != nil {
 		panic(fmt.Sprintf("could not determine if auth is activated: %v", err))
```

Comment:
```suggestion
		tb.Fatal(fmt.Sprintf("could not determine if auth is activated: %v", err))
```

Not actually an issue this pr introduced, but it's probably more idiomatic to call `Fatal` in a test than to panic.